lib/Dialect/AIEVec/Transforms/ConvertVectorToAIEVec.cpp

```diff
@@ -173,6 +173,32 @@ static bool canFoldAIEShiftAndBroadcast(aievec::BroadcastOp op,
   return true;
 }
 
+static aievec::MulElemOp createMulElemOp(ConversionPatternRewriter &rewriter,
```

Comment:
A less generic name and a short description of what it does is preferable. Like "inserts an aievec::MulElemOp at specified location together with glue ops to conform to the signature of the hardware intrinsic". Can't think of a good name for this right now, but "createMulElemOp" somewhat obscures the purpose of the function.